src/utils/toggleSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const toggleSlice = createSlice({
+  name: "toggle",
+  initialState: {
+    isMenuOpen: true,
+  },
+  reducers: {
+    toggleMenu: (state) => {
```

Comment:
```suggestion
    setToggleMenu: (state) => {
```

Reply:
but you are setting the Toggle so we basically follow: gettter/setter function convention

getFormData
setFormData